### PR TITLE
AUI-2035 AUI Button should have an option to don't add  .btn-default and also .btn default classes

### PR DESCRIPTION
**Pull request by fernandosouza**

### CODE REVIEW

**Comment by thiago-rocha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-button/js/aui-button-core.js [hunk 1]
```diff
@@ -50,11 +51,23 @@ ButtonExt.ATTRS = {
...
+    },
+
+    /**
+     * Defines if the button will discard the default button classes.
+     *
+     * @attribute discardDefaultButtonCssClasses
+     * @default false
+     * @type {Boolean}
+     */
+    discardDefaultButtonCssClasses: {
```
Comment (thiago-rocha) [inline src/aui-button/js/aui-button-core.js]: You don't add attribute change cycle (afterChange/uiSet....), if you sure about this decision please add the writeOnce to this attribute.

File: src/aui-button/js/aui-button-core.js [hunk 2]
```diff
@@ -157,6 +171,24 @@ ButtonExt.prototype = {
...
     /**
+     * Includes default button classes if necessary.
+     * Fires after `renderUI` method. 
+     *
+     * @method renderButtonExtUI
+     * @protected
+     */
+    renderButtonExtUI: function() {
+        var instance = this,
+            node = this.getNode(),
```
Comment (thiago-rocha) [inline src/aui-button/js/aui-button-core.js]: This var is never used.

**Comment by thiago-rocha** [comment]
Pull request submitted to https://github.com/fernandosouza/alloy-ui/pull/8. [See changes here.](https://github.com/thiago-rocha/alloy-ui/compare/541c922~1...541c922) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: src/aui-button/js/aui-button-core.js (hunk 1 above)
Comment (fernandosouza) [inline src/aui-button/js/aui-button-core.js]: Yeah. The idea is this attribute to be written once in initialization.

File: src/aui-button/js/aui-button-core.js (hunk 2 above)
Comment (fernandosouza) [inline src/aui-button/js/aui-button-core.js]: It's strange. I haven't this variable here.

**Pull request closed without merging**